src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -119,5 +185,353 @@ class QBridgeMIBObjects(metaclass=MIBMeta, prefix='.1.3.6.1.2.1.17.7.1'):
+                data = self.vlan_untag_ports_map.get(sub_id, None)
+                if data:
+                    byts = data.tobytes()
+                    bs = byts[0:self.vlan_max_port[sub_id[1]]//8 +2]
+                    bs_hex = " ".join([format(i, '02x') for i in bs])
+                    return bs_hex
+            else:
+                return '0'
+
+    def dot1q_vlan_status(self, sub_id):
+        if sub_id:
+            return Dot1qVlanStatusConst.permanent
+
+    def get_vlan_deletes(self):
+        return 0
+
+    def get_next(self, sub_id):
+        right = bisect_right(self.vlan_time_index_list, sub_id)
+        if right == len(self.vlan_time_index_list):
+            return None
+
+        return self.vlan_time_index_list[right]
+
+class Dot1qVlanCurrentMIBObjects(metaclass=MIBMeta, prefix='.1.3.6.1.2.1.17.7.1'):
+    """
+    'Forwarding Database' https://tools.ietf.org/html/rfc4363
+    """
+
+    fdb_vlan_updater = Dot1qVlanCurrentUpdater()
+
+    dot1qVlanNumDeletes = \
+        MIBEntry('4.1.0', ValueType.COUNTER_32, fdb_vlan_updater.get_vlan_deletes)
+
+    dot1qVlanCurrentEgressPorts = \
+        SubtreeMIBEntry('4.2.1.4', fdb_vlan_updater, ValueType.OCTET_STRING, fdb_vlan_updater.dot1q_vlan_current_egress_ports)
+    dot1qVlanCurrentUntaggedPorts = \
+        SubtreeMIBEntry('4.2.1.5', fdb_vlan_updater, ValueType.OCTET_STRING, fdb_vlan_updater.dot1q_vlan_current_untag_ports)
+    dot1qVlanStatus = \
+        SubtreeMIBEntry('4.2.1.6', fdb_vlan_updater, ValueType.INTEGER, fdb_vlan_updater.dot1q_vlan_status)
+
+class Dot1qVlanRowStatusConst:
+    active = 1
+    notInService = 2
+
+class Dot1qVlanStaticUpdater(Dot1qVlanCurrentUpdater):
+    def __init__(self):
+        super().__init__()
+        self.vlan_static_egress_ports_map = {}
+        self.vlan_static_untag_ports_map = {}
+        self.vlan_name_map = {}
+        self.vlan_max_port = {}
+        self.vlan_index_list = []
+
+    def update_data(self):
+        """
+        Update redis (caches config)
+        Pulls the table references for each interface.
+        """
+        fdb_vlan = Dot1qVlanCurrentMIBObjects.fdb_vlan_updater
+        self.vlan_static_egress_ports_map = fdb_vlan.vlan_static_egress_ports_map
+        self.vlan_static_untag_ports_map = fdb_vlan.vlan_static_untag_ports_map
+        self.vlan_name_map = fdb_vlan.vlan_name_map
+        self.vlan_max_port = fdb_vlan.vlan_max_port
+        self.vlan_index_list = fdb_vlan.vlan_index_list
+
+    def dot1q_vlan_static_egress_ports(self, sub_id):
+        if sub_id:
+            if sub_id[0] in self.vlan_static_egress_ports_map:
+                data = self.vlan_static_egress_ports_map.get(sub_id[0], None)
+                if data:
+                    byts = data.tobytes()
+                    bs = byts[0:self.vlan_max_port[sub_id[0]]//8 +2]
+                    bs_hex = " ".join([format(i, '02x') for i in bs])
+                    return bs_hex
+            else:
+                return '0'
```

Comment:
could define the default return value instead of using a magic string.

Reply:
I have defined ZERO_STRING = '0' and using this above. I hope this is what you are expecting.